src/functional/ffront/common_types.py

```diff
@@ -69,7 +69,7 @@ class OffsetType(SymbolType):
     target: Optional[tuple[func_common.Dimension, ...]] = None
 
     def __str__(self):
-        return f"Offset[{self.id}]"
```

Comment:
This must have been some copy pasta bug or a leftover (did this class ever have an `id` attribute?).